tests/ci/json_helper.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
```

Comment:
I am not familiar with json, unfortunately, hence my probably dumb question: since you are wrapping some of the json logic in Python, wouldn't it be possible to do everything in Python?

Reply:
Thanks for the explanation. I don't know and understand json, and it seems complicated to use multiple nested languages. But if you are comfortable and familiar with it, then that's fine.

Reply:
Bash has `jq` tool to handle json. But it needs to be installed, so Python 3 was chosen instead. Maybe in the future, Python 3 will handle GitHub API, thus simplifying `manage.yml`

Reply:
The main question here is do we need to do all of this. Can't we just have a simple action file that runs a container and in that container we run a script that in turn runs ci.sh which executes whatever needs to be executed. Simple as that. No messing with GitHub API, no parsing ci.test in yml, no creation of json, no parsing of that json in python, no manage.yml, nothing. Just a shell script that runs in a container that returns success of fail exit code. That's all.

Reply:
I recall @DusanJovic-NOAA did not want GitHub Actions to be fired up every time a commit is made during our typical PR process, where multiple commits are common before a PR is merged. So, I addressed that issue with GitHub API and extra yml file and python script.

Reply:
Exactly. That's why I think these kind of tests (tests that run longer than few minutes) are simply not suitable for these "continuous integration" testing environments.